meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-docker.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Docker key.json location
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=docker.service
```

Comment:
I'm wondering if it would not be better to have docker service depend on this mount service. Cause otherwise it will fail.